### PR TITLE
Add swagger doc

### PR DESCRIPTION
**Pull request by abdulbasitkay**
My attempt at #59, hope this works

### CODE REVIEW

File: server/docs/definitions.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        enum: [production, test]
+        example: test
+      eventTime:
+        type: string
+        format: unix-time
+        description: valid timestamp of when event is triggered
+        example: 1522767185
+      eventData:
+        type: object
+        description: raw data of the event being streamed
```
Comment (andrewmarklloyd) [inline server/docs/definitions.yaml:36]: The eventData maps to a specific eventType's required keys. For example the [`playEpisode` eventType](https://github.com/SoftwareEngineeringDaily/sedaily-devops/blob/develop/config/param-validation.js#L38) eventData has 3 fields: episodeName, minutesPlayed, and minutesRemaining. I would add an example, like:

```
example: {
    episodeName: "Ethereum Governance with Hudson Jameson",
    minutesPlayed: 1622.53,
    minutesRemaining: 117.62
}
```

**Comment by andrewmarklloyd** [comment]
Looking good so far @abdulbasitkay! I started a review and I have a few notes here:
- We've disabled authentication for now, but I think it is a good idea to keep the documentation so nice work.
- I don't think we need the `server/docs/parameters.yaml` file since none of the endpoints support it. I assume this was a copy over from the other API project?
- I would add docs for the `server/routes/event.route.js` as well. That route doesn't have quite the same validation yet but after [issue 57](https://github.com/SoftwareEngineeringDaily/sedaily-devops/issues/57) it should

**Comment by abdulbasitkay** [comment]
Thanks for the feedback @andrewmarklloyd,
I didn't get your last point. I've already added some docs to `server/routes/event.route.js` or were you referring to `server/routes/error.route.js` that's mentioned in issue #57?

**Comment by andrewmarklloyd** [comment]
Sorry about that, you're right I meant `error.route.js`

**Comment by andrewmarklloyd** [comment]
Let's merge this in and we can add a new issue to add docs for `error.route.js`

**Review by andrewmarklloyd — CHANGES_REQUESTED** [review]

**Review by andrewmarklloyd — APPROVED** [review]

**Comment by andrewmarklloyd** [comment]
Created a new issue #73

**Pull request merged**